### ISSUE TITLE
Combining `aws-cf-modules` and `cfn-modules`

### ISSUE THREAD

**Issue by ambsw-technology** [cfn-modules/docs#34]
TL;DR we're creating [shims](https://github.com/ambsw/cfn-modules-shims) that make it possible to use the two systems together.

We started on `aws-cf-modules` because we were new to CF and needed features (e.g. [cloudmap for ecs](https://github.com/cfn-modules/fargate-service/issues/18#issuecomment-537088767)) that were only found there.  We have finally reached the point that we need features (e.g. `nlb`) not found there,  When possible, I decided to use `cfn-modules` so we can benefit from new features on either side of the fence.

Since the interfaces for the two systems differ, I started creating [shims](https://github.com/ambsw/cfn-modules-shims) that allow an object created in one of these packages to be used in another.  I know this isn't a traditional issue with `cfn-moduels`, but figured it would help future users find the shims.

We'd be willing to contribute the code to either library if there's any interest.  For example, the `vpc` module could be updated to include `module.yml`, `from-aws-cf-modules.yml`, and `to-aws-cf-modules.yml`.

**Comment by michaelwittig** [comment]
Let's see how often the shims are used. If this is a use case that many people are faced with I'm happy to include them in the modules directly. Thanks @ambsw-technology
